plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -431,25 +492,27 @@ fu_pxi_device_write_firmware (FuDevice *device,
 static gboolean
 fu_pxi_device_fw_get_info (FuPxiDevice *self, GError **error)
 {
-	guint8 req[64] = { 0x0 };
+	guint8 res[FU_PXI_DEVICE_OTA_BUF_SZ];
```

Comment:
I think setting this explicitly to NUL using  = { 0x0 } is a good idea personally...